modules/@apostrophecms/asset/index.js

```diff
@@ -22,12 +23,21 @@ module.exports = {
       ...globalIcons
     };
   },
+  handlers (self) {
+    return {
+      'apostrophe:modulesReady': {
+        async runUiBuildTask() {
+          await self.apos.task.invoke('@apostrophecms/asset:build');
```

Comment:
This would run it all the time...?

Reply:
OK I see, but it should pass an option (which will arrive in argv) signifying that the "only if you think it's necessary" behavior is wanted.